src/everest/version.py

```diff
@@ -0,0 +1,17 @@
+# file generated by setuptools_scm
+# don't change, don't track in version control
```

Comment:
Is this supposed to be here?

Reply:
Good point, I removed everything related to this, and just use the ert version